.config/dotnet-tools.json

```diff
@@ -6,13 +6,22 @@
       "version": "7.0.0",
       "commands": [
         "fantomas"
-      ]
+      ],
+      "rollForward": false
     },
     "fsdocs-tool": {
       "version": "20.0.0",
       "commands": [
         "fsdocs"
-      ]
+      ],
+      "rollForward": false
+    },
+    "dotnet-outdated-tool": {
```

Comment:
add useful dotnet tool to run and preview package update from cli, use -inc to only unclude specific namespaces parts and -u to execute update